plugins/arDominionPlugin/css/less/bootstrap-mod.less

```diff
@@ -27,3 +27,8 @@
 .open > .dropdown-menu {
   z-index: 10004
 }
+
+// Upper padding for alerts
+.alert {
+  margin-top: 1em
```

Comment:
(I ended up just getting rid of the ID and using app-alert as that's probably sufficient granularity for our needs.)